verticapy/tests_new/conftest.py

```diff
@@ -523,3 +527,63 @@ def airline_vd_fun(schema_loader):
     airline = load_airline_passengers(schema_loader, "airline")
     yield airline
     drop(name=f"{schema_loader}.airline")
+
+
+@pytest.fixture(scope="module")
+def pop_growth_vd(schema_loader):
+    """
+    Create a dummy vDataFrame for load_pop_growth dataset
+    """
+    pop_growth = load_pop_growth(schema_loader, "pop_growth")
+    yield pop_growth
+    drop(name=f"{schema_loader}.pop_growth")
+
+
+@pytest.fixture(scope="function")
+def pop_growth_vd_fun(schema_loader):
```

Comment:
@vikash018 Any particular reason for this scope to be function? if it is not required, then you can make it broader. It will increase the speed.